nuxeo-page-provider.html

```diff
@@ -265,6 +265,11 @@
               notify: true,
               readOnly: true,
             },
+
+            /**
+             * If `true`, aggregagtes from page provider definition will not be computed.
+             */
+            skipAggregates: Boolean,
```

Comment:
Should we rename `fetchAggregates ` to `fetchAggregateKeys`, then? 😬

Reply:
Why not.. should we keep backward compat?